src/view.rs

```diff
@@ -179,6 +179,21 @@ impl<IV: IntoView + 'static> IntoView for Vec<IV> {
     }
 }
 
+/// Default implementation of `View::layout()` which can be used by
+/// view implementations that need the default behavior and also need
+/// to implement that method to do additional work.
+pub fn recursively_layout_view(id: ViewId, cx: &mut LayoutCx) -> NodeId {
```

Comment:
The view module is private so this function is inaccessible from user code

Reply:
Ugh. I'll export it.

Reply:
Fixed in b2ea0ae2a36a411cd9c69ccbe860c9978a7a72e7